build.cake

```diff
@@ -137,15 +145,26 @@ Task("Copy-Files")
     .IsDependentOn("Run-Unit-Tests")
     .Does(() =>
 	{
-		EnsureDirectoryExists(parameters.ResultBinDir);
-
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.dll", parameters.ResultBinDir);
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.pdb", parameters.ResultBinDir);
-		CopyFileToDirectory(parameters.BuildDir + "/Cake.SqlServer.xml", parameters.ResultBinDir);
-
-		CopyFiles(parameters.BuildDir + "/Microsoft.*.dll", parameters.ResultBinDir);
-
-		CopyFiles(new FilePath[] { "LICENSE", "README.md", "ReleaseNotes.md" }, parameters.ResultBinDir);
+        EnsureDirectoryExists(parameters.ResultBinDir);
+        EnsureDirectoryExists(parameters.ResultBinDir + "/net46");
+        EnsureDirectoryExists(parameters.ResultBinDir + "/netstandard2.0");
+        EnsureDirectoryExists(parameters.ResultBinDir + "/net5.0");
+
+        CopyFileToDirectory(parameters.BuildDir + "/net46/Cake.SqlServer.dll", parameters.ResultBinDir + "/net46");

```

Comment:
Do we still need to build for 4.6? (I've been out of touch with Cake for a while and not sure if that is still required)

Reply:
main intention was that Cake is also still build against net46 - https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Cake.Core.csproj - I thought that in cases where in combination with other plugins someone will need to use full framework then this can help - anyway please advise